scripts/dataproc-workflow-composer/clouddq_composer_dataplex_task_job.py

```diff
@@ -0,0 +1,226 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import datetime
+
+from airflow import models
+from airflow.operators.bash import BashOperator
+from airflow.operators.python import BranchPythonOperator
+
+from airflow.providers.google.cloud.operators.dataplex import (
+    DataplexCreateTaskOperator,
+    DataplexDeleteTaskOperator,
+    DataplexGetTaskOperator,
+    DataplexListTasksOperator,
+)
+from airflow.providers.google.cloud.sensors.dataplex import DataplexTaskStateSensor
+
+import google.auth
+from requests_oauth2 import OAuth2BearerToken
+import json
+import requests
+import time
+
+DATAPLEX_PROJECT_ID = "your-dataplex-project-id"  # The Google Cloud Project where the Dataplex task will be created
+DATAPLEX_REGION = "your-dataplex-region-id"  # The region of the Dataplex Lake where the data quality task will be created.
+DATAPLEX_LAKE_ID = "your-dataplex-lake-id"  # dataplex lake id
+SERVICE_ACC = "service-account-to-execute-task"  # The service account used for executing the task. Ensure this service account has sufficient IAM permissions on your project including BigQuery Data Editor, BigQuery Job User, Dataplex Editor, Dataproc Worker, Service Usage Consumer
+PUBLIC_CLOUDDQ_EXECUTABLE_BUCKET_NAME = "your-public-bucket-with-clouddq-executable-and-hashsum" #Public Cloud Storage bucket containing the prebuilt data quality executable artifact and hashsum. There is one bucket per GCP region.
```

Comment:
"#Public": missing space between "#" and "Public"

Reply:
updated